Packs/ProofpointThreatResponse/Integrations/ProofpointThreatResponse/ProofpointThreatResponse.py

```diff
@@ -494,38 +545,44 @@ def fetch_incidents_command():
     """
     integration_params = demisto.params()
     last_fetch = demisto.getLastRun().get('last_fetch', {})
+    already_fetched = demisto.getLastRun().get('already_fetched', [])
+
+    fetch_delta = integration_params.get('fetch_delta', '6')
+    fetch_limit = integration_params.get('fetch_limit', '50')
+
     incidents_states = integration_params.get('states')
     for state in incidents_states:
         if not last_fetch.get(state):
             last_fetch[state] = FETCH_TIME
 
     incidents = []
-
+    new_fetched_ids = []
     for state in incidents_states:
         request_params = {
             'created_after': last_fetch[state],
-            'state': state
+            'fetch_delta': fetch_delta,
+            'state': state,
+            'already_fetched': already_fetched,
+            'fetch_limit': fetch_limit
         }
 
-        state_parsed_fetch = datetime.strptime(last_fetch[state], TIME_FORMAT)
-        incidents_list = get_incidents_batch_by_time_request(request_params)
-        filtered_incidents_list = filter_incidents(incidents_list)
```

Comment:
Why did you remove the filter incidents?

Reply:
it has been moved to the `get_incidents_batch_by_time_request` function to make it easier to limit the results after adding fetch_limit.